readme.md

```diff
@@ -8,44 +8,53 @@ This repo contains the [Weaviate](https://weaviate.io) website, it's built using
 
 - Software Dependencies
   - [Ruby](https://www.ruby-lang.org/en/documentation/installation/)
-  - Ruby Development Package (aka ruby-dev)
 
 - Installing Dependencies in **Ubuntu/Debian**
   ```bash
-  # update repositories & install basic build dependencies
-  sudo apt update && sudo apt install -y build-essential bash git rsync
-
-  # update repositories & install rbenv
-  sudo apt update && sudo apt install rbenv ruby-build ruby_dev
-
-  # initialize rbenv
-  rbenv init
-
-  # updates ruby-build local packages
-  git clone https://github.com/rbenv/ruby-build.git "$(rbenv root)"/plugins/ruby-build
-
-  # install ruby 2.7.5
-  rbenv install 2.7.5
-
-  # initialize rbenv
-  export PATH="$HOME/.rbenv/bin:$PATH"
+  # update
+  sudo apt update
+  
+  # install dependencies for installing ruby
+  sudo apt install git curl libssl-dev libreadline-dev zlib1g-dev autoconf bison build-essential libyaml-dev libreadline-dev libncurses5-dev libffi-dev libgdbm-dev
```

Comment:
Also I think we don't need most of the dev packages `libssl-dev libreadline-dev libyaml-dev libreadline-dev libncurses5-dev libffi-dev libgdbm-dev` because Ruby works fine without them.

There are certain issues regarding these packages, you can refer rbenv's GitHub. The pre-installed packages with Ubuntu/Debian should work just fine I guess.

Reply:
I followed this page to install ruby and it worked as opposed to the already given steps.

[Digitial Ocean How to Install ruby](https://www.digitalocean.com/community/tutorials/how-to-install-ruby-on-rails-with-rbenv-on-ubuntu-20-04)

If you think, some of the steps might be redundant, then do let me know.